crates/types/src/traits/node_implementation.rs

```diff
@@ -314,13 +337,24 @@ where
             entry.clone(),
             sk.clone(),
         );
-        let committee_exchange =
-            COMMITTEEEXCHANGE::create(entries, configs.1, networks.1, pk, entry, sk);
+
+        let committee_exchange = COMMITTEEEXCHANGE::create(
+            entries.clone(),
+            configs.1.clone(),
+            networks.1,
+            pk.clone(),
+            entry.clone(),
+            sk.clone(),
+        );
+
+        // RM TODO: figure out if this is the proper config
+        let vid_exchange = VIDEXCHANGE::create(entries, configs.1, networks.3, pk, entry, sk);
```

Comment:
I believe this is the correct config for the exchange, because it uses the same one VID uses originally (CommitteeExchange). I also tried with the other configurations but it had some odd results. I'll make an issue to investigate though

Reply:
Maybe @bfish713 knows the answer.